shiny/playwright/controller/_accordion.py

```diff
@@ -98,18 +98,18 @@ def expect_body(self, value: PatternOrStr, *, timeout: Timeout = None) -> None:
         """
         playwright_expect(self.loc_body).to_have_text(value, timeout=timeout)
 
-    def expect_icon(self, value: PatternOrStr, *, timeout: Timeout = None) -> None:
+    def expect_icon(self, value: bool, *, timeout: Timeout = None) -> None:
```

Comment:
We should rename `value` to `exists` as it is not the value submitted by app author